### ISSUE TITLE
notifications_notification_details_opened being tracked on every app launch

### ISSUE THREAD

**Issue by etoledom** [wordpress-mobile/WordPress-iOS#11975]
### Expected behavior
`notifications_notification_details_opened` should not be tracked when the user does not navigate to that section.

### Actual behavior

Every time I launch the app, I see this tracks event:
```
🔵 Tracked: notifications_notification_details_opened, properties: {
    "notification_type" = comment;
}
```
Even when the app starts in `My Sites` tab, and the `Notifications` tab is in the root (not showing any notification details).

### Steps to reproduce the behavior

Launch the app from Xcode.
Check that the Track event log mentioned above is there.

**Comment by etoledom** [comment]
@designsimply - Not sure about the priority of this, but track events are sensible matter.

**Comment by Gio2018** [comment]
Closing this one as it was also addressed by #16652
cc @startuptester
